src/components/CustomAppDetails/Versions.js

```diff
@@ -0,0 +1,271 @@
+import { useAlert, useConfig } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { PropTypes } from '@dhis2/prop-types'
+import {
+    Checkbox,
+    SingleSelectField,
+    SingleSelectOption,
+    Button,
+    Table,
+    TableHead,
+    TableRowHead,
+    TableCellHead,
+    TableBody,
+    TableRow,
+    TableCell,
+} from '@dhis2/ui'
+import moment from 'moment'
+import React, { useState } from 'react'
+import semver from 'semver'
+import { useApi } from '../../api'
+import { channelToDisplayName } from './channel-to-display-name'
+import styles from './CustomAppDetails.module.css'
+
+const ChannelCheckbox = ({
+    name,
+    label,
+    channelsFilter,
+    setChannelsFilter,
+}) => {
+    const handleChange = ({ checked }) => {
+        const newState = new Set(channelsFilter)
+        if (checked) {
+            newState.add(name)
+        } else {
+            newState.delete(name)
+        }
+        setChannelsFilter(newState)
+    }
+
+    return (
+        <div className={styles.channelCheckbox}>
+            <Checkbox
+                checked={channelsFilter.has(name)}
+                disabled={channelsFilter.size === 1 && channelsFilter.has(name)}
+                onChange={handleChange}
+                label={label}
+            />
+        </div>
+    )
+}
+
+ChannelCheckbox.propTypes = {
+    channelsFilter: PropTypes.object.isRequired,
+    label: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    setChannelsFilter: PropTypes.func.isRequired,
+}
+
+const Filters = ({
+    versions,
+    channelsFilter,
+    setChannelsFilter,
+    dhisVersionFilter,
+    setDhisVersionFilter,
+    dhisVersions,
+}) => {
+    const hasChannel = channel => versions.some(v => v.channel === channel)
+
+    return (
+        <div className={styles.versionsFilters}>
+            <h3 className={styles.sectionSubheader}>
+                {i18n.t('Channel', { context: 'AppHub release channel' })}
+            </h3>
+            {Object.keys(channelToDisplayName)
+                .filter(hasChannel)
+                .map(name => (
+                    <ChannelCheckbox
+                        key={name}
+                        name={name}
+                        label={channelToDisplayName[name]}
+                        channelsFilter={channelsFilter}
+                        setChannelsFilter={setChannelsFilter}
+                    />
+                ))}
```

Comment:
I don't know that we should show this here... we don't want to promote development and canary builds to users who don't really know what they mean.  I think since we only have stable builds at the moment we should omit this set of checkboxes entirely.